src/component/Levels.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import Button from "./Button";
+
+function Levels({ levelSetUp, easy, medium, hard }) {
+  const [customeLevel, setcustomeLevel] = useState(easy);
```

Comment:
What's need of level here?

We can psss it from parent since we are storing it there & also updating it